### PR TITLE
fix(tabs): add dark mode variables

### PR DESCRIPTION
**Pull request by EmelieLitwin**
**Describe pull-request**  
Dark mode variables

**Solving issue**  
Fixes: [AB#2936](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2936)

### CODE REVIEW

**Review by alexandereneroth — CHANGES_REQUESTED** [review]
I think the foreground of the scroll buttons should be some shade of white, hard to spot otherwise (inline-tabs, and inline-tabs-fullbleed)
Screenshot 2023-01-03 at 10 45 33

**Review by mJarsater — APPROVED** [review]
LGTM. The color of the disabled state for the inline tabs is very similar to the non disabled one, might be an a11y issue. But that's a design thing. Nice work! 🦄

File: tegel/src/components/tabs/inline-tabs-fullbleed/inline-tabs-fullbleed-theme-vars.scss [hunk 1]
```diff
@@ -1,21 +1,47 @@
-:root,
+:root,-hover--sdds-inline-tabs-fullbleed-tab-color-opacity-disabledleed-tab-color-disabled-opacityinline
```
Comment (alexandereneroth) [inline tegel/src/components/tabs/inline-tabs-fullbleed/inline-tabs-fullbleed-theme-vars.scss:1]: Looks like an unintentional find/replace mistake here

File: tegel/src/components/tabs/inline-tabs-fullbleed/inline-tabs-fullbleed-theme-vars.scss [hunk 2]
```diff
@@ -1,21 +1,47 @@
...
 .sdds-theme-light {
```
Comment (alexandereneroth) [inline tegel/src/components/tabs/inline-tabs-fullbleed/inline-tabs-fullbleed-theme-vars.scss:2]:
```suggestion
:root,
.sdds-theme-light {
```

**Review by alexandereneroth — CHANGES_REQUESTED** [review]

**Review by alexandereneroth — APPROVED** [review]

**Pull request merged**